Symfony/CS/Console/Command/FixCommand.php

```diff
@@ -529,18 +530,37 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 throw new \InvalidArgumentException(sprintf('The format "%s" is not defined.', $input->getOption('format')));
         }
 
-        if (!$this->errorsManager->isEmpty()) {
-            $output->writeLn('');
-            $output->writeLn('Files that were not fixed due to internal error:');
+        $lintErrors = $this->errorsManager->getLintingErrors();
+        if (!empty($lintErrors)) {
+            $this->listErrors($output, 'linting', $lintErrors);
+        }
 
-            foreach ($this->errorsManager->getErrors() as $i => $error) {
-                $output->writeLn(sprintf('%4d) %s', $i + 1, $error['filepath']));
-            }
+        $fixErrors = $this->errorsManager->getFixingErrors();
+        if (!empty($fixErrors)) {
+            $this->listErrors($output, 'fixing', $fixErrors);
         }
 
         return !$resolver->isDryRun() || empty($changed) ? 0 : 3;
```

Comment:
Can't see how using the `--dry-run` option would change anything in my opinion.
But that is only my personal opinion, if more people will want to treat it as an failure we could make it so.